requirements.txt

```diff
@@ -19,4 +19,4 @@ pytest-ordering~=0.6
 sphinx-jsonschema~=1.16
 pytest-console-scripts~=1.1
 pre-commit==2.10.1
-scipy
+scipy
```

Comment:
The `requirements.txt` conflict is because of this change, which looks like a newline/whitespace change.

(In other words, either a newline was removed at the end of the file, or a newline was added at the end of the file.)

Reply:
Great, thanks for the response. fixing now.